src/styles/clock.scss

```diff
@@ -0,0 +1,62 @@
+.clock {
+  width: 400px;
+  height: 400px;
+  border: 15px solid #2f4f4f;
+  border-radius: 50%;
+  background: white;
+
+  transition: all 1s ease-in-out;
+
+  &:hover {
+    box-shadow: 0 0 100px;
+  }
+
+  &:before {
+    content: "";
+    position: absolute;
+    top: calc(50% - 3px);
+    left: 50%;
+    height: 7px;
+    width: 180px;
+    background: red;
+    border-radius: 5px;
+    animation: animate 60s linear infinite;
+    transform-origin: left;
+  }
+
+  &:after {
+    content: "";
+    position: absolute;
+    top: calc(50% - 3px);
+    left: 50%;
+    height: 10px;
+    width: 140px;
+    background: black;
+    border-radius: 5px;
+    animation: animate 3600s linear infinite;
+    transform-origin: left;
+  }
```

Comment:
before and after have almost same styles.
make mixin for them with necessary parameters

Reply:
tell me why this mixin does not work in a separate file?! if for example move "./block/clock.scss" and mixin "./utils/mixin.scss".

Reply:
i include @import "./block/clock.scss"; and @import "./utils/mixin.scss"; file stops working!

Reply:
good?